babel.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  presets: [['@babel/preset-env', { targets: { node: 'current' } }]]
+};
```

Comment:
Feels like much of this commit could be merged into previous commits as fixes.

Reply:
Taking a guess, but I'm assuming this means you'd rather the lint fixes be put onto the original commit instead?

Reply:
That would be correct.

Reply:
If you have ever had to do a large bisect to try to find a bug then you understand why you get twitchy that every patch should be correct and stand alone.  I would guess we are likely to never have to do that here... but I have a lot in other code bases and it really has burned into my being that need for patches to be atomically as correct as can be.